it/google-cloud-platform/src/main/java/com/google/cloud/teleport/it/gcp/dataflow/DirectRunnerClient.java

```diff
@@ -39,7 +38,7 @@
  * Implementation of the {@link PipelineLauncher} interface which invokes the template class using
  * DirectRunner, and manages the state in-memory.
  */
-public class DirectRunnerClient extends AbstractPipelineLauncher {
+public class DirectRunnerClient extends AbstractDataflowPipelineLauncher {
```

Comment:
This is not DataflowPipelineLauncher. I think AbstractPipelineLauncher captures all cases here

Reply:
Changed back to AbstractPipelineLauncher. Was considering AbstractDataflowPipelineLauncher because I see getJob, drainJob, cancelJob methods all return a `com.google.api.services.dataflow.model.Job` object.